examples/emodel_pipeline_local_python/monitor_optimisation.py

```diff
@@ -27,7 +27,7 @@ def monitor_optimisation():
     emodel = "L5PC"
     species = "rat"
     brain_region = "SSCX"
-
+    continue_opt = True # if set to False, will show the optimisation status of the last checkpoint
```

Comment:
I don't think this comment is correct. If set to `False`, it will consider that the existence of a checkpoint means that the optimisation status is finished.
Also we kind of always want this to be True in this file, so we could hardcode `continue_opt=True` in the `access_point.optimisation_state` function call and don't have any `continue_opt` variable in this file. What do you think?